Spigot-Server-Patches/0010-Adventure.patch

```diff
@@ -170,18 +170,18 @@ index 0000000000000000000000000000000000000000..75c36e772be93c9808cd8b72410ad8ba
 +            // continuing from AsyncPlayerChatEvent (without PlayerChatEvent)
 +            event -> {
 +                this.processModern(
-+                    legacyComposer(event.getFormat(), legacyDisplayName((CraftPlayer) event.getPlayer()), event.getMessage()),
++                    legacyComposer(event.getFormat()),
 +                    event.getRecipients(),
-+                    PaperAdventure.LEGACY_SECTION_UXRC.deserialize(event.getMessage()),
++                    PaperAdventure.LEGACY_SECTION_UXRC.deserialize(event.getMessage()).replaceText(URL_REPLACEMENT_CONFIG),
```

Comment:
Not really sure if these are needed, since replacement will be done in legacyComposer later anyway.